spec/pom.xml

```diff
@@ -16,24 +16,38 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.eclipse.ee4j.jms</groupId>
         <artifactId>jms-parent</artifactId>
         <version>3.0-SNAPSHOT</version>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
+
     <artifactId>messaging-spec</artifactId>
     <packaging>pom</packaging>
+
     <name>Jakarta Messaging Specification</name>
 
+    <distributionManagement>
+      <repository>
+        <id>localhost</id>
+        <url>file://${basedir}/target/repo/</url>
+      </repository>
+      <snapshotRepository>
+        <id>localhost</id>
+        <url>file://${basedir}/target/snapshot-repo/</url>
+      </snapshotRepository>
+    </distributionManagement>
+
     <properties>
         <site.output.dir>${project.build.directory}/staging</site.output.dir>
         <maven.site.skip>true</maven.site.skip>
-        <asciidoctor.maven.plugin.version>1.5.7.1</asciidoctor.maven.plugin.version>
-        <asciidoctorj.version>1.6.2</asciidoctorj.version>
-        <asciidoctorj.pdf.version>1.5.0-alpha.16</asciidoctorj.pdf.version>
-        <jruby.version>9.2.6.0</jruby.version>
+        <asciidoctor.maven.plugin.version>2.0.0-RC.1</asciidoctor.maven.plugin.version>
+        <asciidoctorj.version>2.2.0</asciidoctorj.version>
+        <asciidoctorj.pdf.version>1.5.0-rc.2</asciidoctorj.pdf.version>
+        <jruby.version>9.2.9.0</jruby.version>
```

Comment:
What motivated the update of all these versions?  Do they fix some specific bugs?

Reply:
The plug-in seems to have issues on JDK 11, the motivation was to use the latest to be as close as possible to a fix for JDK 11.

It's still not 100% solved as there's still some warnings, but, IMHO, it's a step closer.

Reply:
Are there failures or just warnings on JDK 11?

Do the version update reduce the number of warnings?

Reply:
They are warnings. The issue seems to be known with (at least) the JRuby devs as other projects (other than ascii doctor) have run into the same warning. I have hope this will eventually be fixed and that it's therefor a good idea to stay close to the most recent versions.